src/ansi.rs

```diff
@@ -0,0 +1,1625 @@
+//! ANSI Terminal Stream Parsing.
```

Comment:
The name of this file, this documentation and the feature isn't really great. That just describes all of this library basically.